packages/@lightningjs/ui-components/src/components/Shadow/Shadow.d.ts

```diff
@@ -20,8 +20,9 @@ import type lng from '@lightningjs/core';
 import Base from '../Base';
 import type { Color, StylePartial } from '../../types/lui';
```

Comment:
type should be removed

Reply:
I believe this works with `import` or `import type`, but can remove it to be consistent with other TS def files in the repo.